docs/src/main/asciidoc/se/openapi/openapi-generator.adoc

```diff
@@ -23,9 +23,280 @@
 :rootdir: {docdir}/../..
 
 include::{rootdir}/includes/se.adoc[]
+:incdir: {rootdir}/includes/openapi
+:gen-inc: {incdir}/openapi-generator.adoc
+:snippets: {sourcedir}/se/openapi/OpenApiGeneratorSnippets.java
 
-== Coming Soon
+:helidon-client-xref: {webclient-page}
 
-The support for link:{openapi-generator-tool-site-url}[OpenAPI generator] is not available yet.
+include::{gen-inc}[tag=preamble]
```

Comment:
Keep it simple, just use include::{rootdir}/includes/openapi/openapi-generator.adoc

Reply:
changed